vmck/middleware.py

```diff
@@ -0,0 +1,36 @@
+import jwt
+
+from django.http import JsonResponse
+
+from vmck import settings
+
+
+def jwt_authentication_middleware(get_response):
+    def middleware(request):
+        if request.path in settings.UNAUTHENTICATED_PATHS or settings.DEBUG:
+            return get_response(request)
+
+        try:
+            auth_header = request.headers['Authorization']
+            if not auth_header.startswith('Bearer'):
```

Comment:
This looks like a good candidate for regular expressions. It's way more readable and safer than splitting strings.